auction-server/src/opportunity/repository/models.rs

```diff
@@ -22,25 +34,55 @@ pub enum OpportunityRemovalReason {
     Invalid,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct OpportunityMetadataEvm {
```

Comment:
On the api layer what we mean by metadata is opportunity_id, creation time and anything not representing the core opportunity. I think the meaning is reversed here. we should probably use a different word or maybe even nothing?

Reply:
Names in api layer are updated. WDYT?

Reply:
i agree that metadata seems like a weird term to use for the core info about the opportunity; if anything, this is actually the "complement" of the metadata. i lol'd at [this](https://www.powerthesaurus.org/metadata), but maybe `OpportunityContent` would actually be a better term than metadata

Reply:
Yeah, I still think these fields are not metadata

Reply:
I will update it in a separate PR